src/app/core/admin/admin-entity-details/admin-entity-field/configure-entity-field-validator/configure-entity-field-validator.component.html

```diff
@@ -0,0 +1,23 @@
+<form class="validator-form-cell" [formGroup]="form">
+  <mat-tab-group formGroupName="validatorForm">
+    <mat-form-field>
+      <mat-label i18n>Minimum Value</mat-label>
+      <input formControlName="min" matInput type="number" />
+    </mat-form-field>
+
+    <mat-form-field>
+      <mat-label i18n>Maximum Value</mat-label>
+      <input formControlName="max" matInput type="number" />
+    </mat-form-field>
+
+    <mat-form-field>
+      <mat-label i18n>Regex Pattern</mat-label>
+      <input formControlName="regex" matInput />
+    </mat-form-field>
+    <mat-checkbox i18n formControlName="required"> Is Required? </mat-checkbox>
```

Comment:
these checkboxes do not display as a new line yet:
![image](https://github.com/Aam-Digital/ndb-core/assets/1682541/b6e2a05b-0df9-4582-b204-aa96d6649811)

maybe needs to be wrapped in a form-field or div or something?